sale_require_contract/data/data.xml

```diff
@@ -1,16 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<odoo>
-  <data noupdate="1">
+<odoo noupdate="1">
 
-    <record id="require_contract" model="sale.exception">
+    <record id="require_contract" model="exception.rule">
       <field name="name">Contract Required</field>
       <field name="description">You must set a contract.</field>
       <field name="sequence">10</field>
       <field name="model">sale.order</field>
+      <field name="rule_group">sale</field>
       <field name="code">if not object.project_id:
```

Comment:
Hi @ValentinoDefelice thanks for the PR
The field `project_id` not longer exist by this name, now it's call `analytic_account_id` .
Please change the field name here.
Thanks